### ISSUE TITLE
BN is not ok on color constancy task?

### ISSUE THREAD

**Issue by DavideHe** [yanlinqian/Temporal-Color-Constancy#3]
I found many reseaerch about color constancy with CNN use the squeezeNet or AlexNet.And No one use MobileNetv2 or others that is efficient on classification task.
So I tried mobilenetv2 on TCC-net as backbone(mobilenet2 loaded pretrained). But that result is so terrible.
Batch Normalization  is not ok on color constancy task?

**Comment by yanlinqian** [comment]
why bn should work for awb, this is a good question should be explored.
